tests/system/robot/notepadTests.py

```diff
@@ -14,49 +17,289 @@
 from NotepadLib import NotepadLib as _NotepadLib
 from AssertsLib import AssertsLib as _AssertsLib
 import NvdaLib as _NvdaLib
+from robot.libraries.BuiltIn import BuiltIn
 
+_builtIn: BuiltIn = BuiltIn()
 _notepad: _NotepadLib = _getLib("NotepadLib")
 _asserts: _AssertsLib = _getLib("AssertsLib")
 
 
-def test_moveByWord_symbolLevelWord():
-	"""Disabled due to revert of PR #11856 is: "Speak all symbols when moving by words (#11779)
+class Move(_enum.Enum):
+	"""Gestures to move by different amounts"""
+	CHAR = "numpad3"
+	WORD = "numpad6"
+	LINE = "numpad9"
+	HOME = "control+home"
```

Comment:
Since you're using review cursor for navigation it would be much clearer to use Shift+numpad 7 here. Currently this relies on the fact that review cursor follows caret and therefore it is moved to the top of the document and it is better to be explicit about the fact that you want to move review cursor rather than system caret.